src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/tests/Field.test.js

```diff
@@ -447,6 +450,33 @@ test('Call onFinish callback after editing the field has finished', () => {
     expect(finishSpy).toBeCalledWith('/block/0/test', '/test');
 });
 
+test('Call onSuccess callback after editing the field has finished', () => {
```

Comment:
Not if the field has finished, but when the field calls the `onSuccess` callback, right? This call does not happen when the `onFinish` callback is called, that's why it is a bit confusing.